src/solver/optim/impl/LinearProblemImpl.cpp

```diff
@@ -0,0 +1,83 @@
+#include "include/antares/optim/impl/LinearProblemImpl.h"
+#include "ortools/linear_solver/linear_solver.h"
+#include "antares/solver/utils/ortools_utils.h"
+
+using namespace Antares::optim::api;
+using namespace std;
+
+LinearProblemImpl::LinearProblemImpl() = default;
+
+LinearProblemImpl::LinearProblemImpl(bool isMip, const std::string& solverName)
+{
+    this->mpSolver = MPSolverFactory(isMip, solverName);
```

Comment:
so you should not authorize copy constructor, many methods returns ref of this->mpSolver components